src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/DialogRequestedEventArgs.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This Class contains the requests dialog from vc-daemon.
+    /// </summary>
+    public class DialogRequestedEventArgs : EventArgs
+    {
+        internal DialogRequestedEventArgs(int pid, string dispTextString, string uttTextString, bool continuous)
+        {
+            Pid = pid;
+            DisplayText = dispTextString;
+            UtterenceText = uttTextString;
+            IsContinuous = continuous;
+        }
+
+        /// <summary>
+        /// The PID
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int Pid
```

Comment:
Why need pid?

Reply:
Because of the priority.
example, already showing text of "Find it blah blah~" state, Next callback get a "Alarm time" text.
voice-control-manager can choice.
BTW voice-control-maanger see the app's information using pid.

Reply:
I think, AppID better than Pid

Reply:
Yes AppID better than Pid. I agree with you.
but If use getAppId method, There will be a dependency on appControl in voice control manager lib.
https://github.com/Samsung/TizenFX/blob/4173d1fbc061092b86c956d7237daeed3f5b24ac/src/Tizen.Applications.Common/Interop/Interop.AppControl.cs#L63

so voice-control-manager developer uisng getAppId method.

Reply:
Who is give pid information? I think, native code also should be fixed for support appid

Reply:
And, To get AppID from PID, we can use this API
https://github.com/Samsung/TizenFX/blob/f7c1f3b65cd7a6f8a5f86899750f5d49c7ccda64/src/Tizen.Applications.Common/Tizen.Applications/ApplicationManager.cs#L424

Reply:
Pid can be passed by each application which includes voice control client. And native codes have already been published. So we can't change the native c api. And if application developer can get an app id using above api as you mentioned. And also, we don't want to get a dependency of app fw. So, we'd like to keep pid instead of appid.

Reply:
I also can't accept this API

Reply:
You can update native API without ABI break.
so does not matter published native code

Reply:
Changed to AppId. Please check a new patch.